Week1/HTML.md

```diff
@@ -0,0 +1,260 @@
+
+### Task: Build a basic HTML page
+
+- open your favourite editor and create a file called `my-first-page.html`
+- type `html` in the page and press the `tab` key, this will automatically generate an html page for you.
+
+The result should look like the following:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title></title>
+  </head>
+  <body></body>
+</html>
+```
+
+_Note: We will explore all the tags later_
+
+- In the `<body>` tag add an `<h1>`
+
+Headings are defined with the `<h1>` to `<h6>` tags.
+
+`<h1>` defines the most important heading and `<h6>` defines the least important heading.
+
+Your code should look something like this:
+
+```html
+<body>
+  <h1>The Main Heading</h1>
+</body>
+```
+
+- Right click on your html file and click on `open in browser`, this should open the browser with the result of this file. You should see the h1 heading with `The Main Heading` on the page.
+
+<details>
+  <summary>If you are using Atom</summary>
+  To be able to preview your html file in a browser, install the _open-in-browsers_ package by going to:
+  
+  Atom -> Preferences -> Install
+  </details>
+
+- Let's explore this a bit more, in the browser, open the inspector. You can do so by either pressing `f12` or right clicking on the page and then inspect or pressing `ctrl+shift+i`.
+
+  - You should see be able to see the elements you created before (raw html), in the elements tab. You can edit/delete/add elements here like you did in your editor.
+
+  - Change the elements in the browser from the inspector, delete the current `h1` tag and add a `div` tag.
+
+    Find out more about [`div`](https://www.w3schools.com/tags/tag_div.asp)
+
+    _Notice_ that these changes will not be saved in your file and only affect the browser, if you wish to save them then you will have to actually do it from the file itself.
+
+- Add some more elements to in your `div`, after each step check your browser to see the changes:
+
+  - Add the `h1` tag back to your code with the content `The Main Heading`.
+  - Add an `h2` tag that has the content `Sunheading`.  
+  
+    Why are headings [important?](https://www.w3schools.com/html/html_headings.asp)
+  - Add a `p` tag with the content `First paragraph`.
+  - Add an `img` tag with an image from your laptop of your choosing.  
+  
+    More about [`img`](https://www.w3schools.com/tags/tag_img.asp).
+  - Add a second `div` (nested div).
+  - In the second `div` add another `p` tag with the content `Second paragraph`.  
+    
+    Comments are used to leave messages for other devs without affecting the code itself. To add a comment in an html           file use the following syntax: `<!--` your comment goes in here `-->`
+  - Now add a comment to your file just below the main heading.
+  
+  - The result should look something like this (_only use this in case you are stuck_):
+
+    <details>
+     <summary>Code result</summary>
```

Comment:
have this in ```html

so it looks like this

```html
<p>Hello this is an actual html </p>
```

Reply:
You can't add it in the details tag

Reply:
i just tried you can wrap the everything in ````html  some html here> ```